content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -204,54 +211,63 @@ You can read more about removing jobs in [garbage collection](/docs/concepts/arc
-information about working with config files, see [deploying applications](/docs/tasks/run-application/run-stateless-application-deployment/),
+As with all other Kubernetes objects, a CronJob must have `apiVersion`, `kind`, and `metadata` fields.
+For more information about working with Kubernetes objects and their
+{{< glossary_tooltip text="manifests" term_id="manifest" >}}, see the
+[managing resources](/docs/concepts/cluster-administration/manage-deployment/),
 and [using kubectl to manage resources](/docs/concepts/overview/working-with-objects/object-management/) documents.
+Each manifest for a CrobJob also needs a [`.spec`](/docs/concepts/overview/working-with-objects/kubernetes-objects/#object-spec-and-status) section.
 
-A cron job config also needs a [`.spec` section](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
 -->
 ## 编写 CronJob 声明信息 {#writing-a-cronjob-spec}
 
-像 Kubernetes 的其他配置一样，CronJob 需要 `apiVersion`、`kind` 和 `metadata` 字段。
-有关配置文件的一般信息，请参考
-[部署应用](/zh-cn/docs/tasks/run-application/run-stateless-application-deployment/) 和
-[使用 kubectl 管理资源](/zh-cn/docs/concepts/overview/working-with-objects/object-management/)。
+像 Kubernetes 的其他对象一样，CronJob 需要 `apiVersion`、`kind` 和 `metadata` 字段。
+有关 Kubernetes 对象及它们的{{< glossary_tooltip text="清单" term_id="manifest" >}}的更多信息，请参考
+[资源管理](/zh-cn/docs/concepts/cluster-administration/manage-deployment/)和
+[使用 kubectl 管理资源](/zh-cn/docs/concepts/overview/working-with-objects/object-management/)文档。
 
 CronJob 配置也需要包括
-[`.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status)。
+[`.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status) 部分。
 
+{{< note >}}
 <!--
-All modifications to a cron job, especially its `.spec`, are applied only to the following runs.
+If you modify a CronJob, the changes you make will apply to new jobs that start to run after your modification
+is complete. Jobs (and their Pods) that have already started continue to run without changes.
+That is, the CronJob does _not_ update existing jobs, even if those remain running.
 -->
-{{< note >}}
-对 CronJob 的所有改动，特别是它的 `.spec`，只会影响将来的运行实例。
+如果你修改了一个 CronJob，你所做的修改将只被应用到将来所运行的实例上，
+对当前 CronJob 内处于运行中的 Job 集合（和 Job 里面的 Pod）不会产生任何变化，它们将继续运行。
+也就是说，对 CronJob 的修改不更新现有的实例，即使这些实例处于运行状态。
 {{< /note >}}
 
 <!--
 ### Schedule
 
 The `.spec.schedule` is a required field of the `.spec`.
-It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0 * * * *` or `@hourly`, as schedule time of its jobs to be created and executed.
+It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0 * * * *` or `@hourly`,
+as schedule time of its jobs to be created and executed.
+
 -->
-### 时间安排 {#schedule}
+### 调度时间设置 {#schedule}
 
 `.spec.schedule` 是 `.spec` 中的必需字段。它接受 [Cron](https://en.wikipedia.org/wiki/Cron)
```

Comment:
```suggestion
`.spec.schedule` 是 `.spec` 中的必需字段。它接受 [Cron](https://zh.wikipedia.org/wiki/Cron)
```

Reply:
You are really meticulous. Thank you for your review